core-redirects-cae/src/main/resources/META-INF/coremedia/component-core-redirects-cae.properties

```diff
@@ -1,4 +1,7 @@
 # Path to redirect within a site folder
 core.redirects.path=Options/Settings/Redirects
-# Number of threads in the cache key invalidation executor
-core.redirects.cache.parallel.recompute.threads=2
\ No newline at end of file
+
+# Number of threads for complete site index updates
+core.redirects.cache.parallel.site.recompute.threads=4
```

Comment:
für den site Threadpool würde ich eher mehr sehen (hängt natürlich von der Anzahl der Sites im CMS ab) und für den Item Threadpool eher weniger, da reichen bestimmt 1-2.

Vielleicht sollte die Größe vom site Threadpool = Anzahl Sites im CMS sein?

Reply:
Das müssten Kunden dann vermutlich in ihrer Instanz anpassen.
Allerdings ist das hier die maximale Anzahl der Threads. Minimal sind das bei beiden 1. Ich hab die Kommentare entsprechend angepasst.